kafka-admin-client/src/test/java/io/mosip/kafkaadminclient/MosipKafkaAdminClientTest.java

```diff
@@ -0,0 +1,100 @@
+package io.mosip.kafkaadminclient;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.DescribeTopicsResult;
+import org.apache.kafka.clients.admin.KafkaAdminClient;
+import org.apache.kafka.clients.admin.ListTopicsResult;
+import org.apache.kafka.clients.admin.TopicDescription;
+import org.apache.kafka.common.KafkaFuture;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ Admin.class })
+public class MosipKafkaAdminClientTest {
+
+    @SuppressWarnings("unchecked")
+	@Test
+	public void isTopicsPresentTest() throws Exception {
+		PowerMockito.mockStatic(Admin.class);
+		KafkaAdminClient adminMock = mock(KafkaAdminClient.class);
+		PowerMockito.when(Admin.create(any(Properties.class))).thenReturn(adminMock);
+    	ListTopicsResult listTopicsResult = Mockito.mock(ListTopicsResult.class);
+		KafkaFuture<Set<String>> kafkaFuture = Mockito.mock(KafkaFuture.class);
+		Set<String> sampleStrings = new HashSet<>();
+		sampleStrings.add("test");
```

Comment:
In the isTopicPresent method, we do "stream().allMatch(", so can we have multiple items sent here ?

Reply:
Added multiple items.